extensions/wikia/AdEngine/js/slot/service/srcProvider.js

```diff
@@ -18,15 +21,19 @@ define('ext.wikia.adEngine.slot.service.srcProvider', [
 			originalSrc = 'premium';
 		}
 
+		if (babDetection.isBlocking()) {
```

Comment:
+ skin === oasis

Reply:
Right, I added than in IL branch but not here. Fixed.